src/bindings/interface/types.i

```diff
@@ -8,7 +8,7 @@ Only meant to be included, shouldn't be passed to cmake as a module!
 %shared_ptr(libcellml::Component)
 %shared_ptr(libcellml::ComponentEntity)
 %shared_ptr(libcellml::Entity)
-%shared_ptr(libcellml::Error)
+%shared_ptr(libcellml::Issue)
```

Comment:
Move it after `%shared_ptr(libcellml::ImportedEntity)`.

Reply:
Done.